code/datums/abilities/flock/flockmind.dm

```diff
@@ -99,9 +99,9 @@
 /datum/targetable/flockmindAbility/designateEnemy/cast(atom/target)
 	if(..())
 		return 1
-	var/mob/living/M = target
+	var/mob/M = target
 	var/mob/living/intangible/flock/flockmind/F = holder.owner
-	if(isliving(M))
+	if(isliving(M) && !isflock(M))
```

Comment:
maybe also !isintangible(M) so we don't accidentally mark the AI eye as a target.

Reply:
This, but also convert this if into a guard (i.e. early return) instead of it being a block containing all the logic